paper.bib

```diff
@@ -0,0 +1,80 @@
+@misc{HOMER,
+  title = {{HOMER Energy}},
+  year = {2022},
+  howpublished = {\url{https://www.homerenergy.com/}},
+  note = {Accessed: 22 August 2022}
+}
+
+@phdthesis{Sandwell:2017a,
+  author = {Sandwell, Philip},
+  title = {{Assessing the potential for photovoltaic technology to mitigate greenhouse gas emissions:life cycle analysis and energy systems modelling for electricity access in rural India}},
+  publisher = {Imperial College London},
+  year = {2017},
+  address = {{London, UK}}
+}
+
+@misc{Renewables_Ninja,
+  title = {{Renewables.ninja}},
+  howpublished = {\url{https://www.renewables.ninja/}},
+  note = {Accessed: 22 August 2022}
+}
+
+@artcle{Pfenninger:2016,
+  author = {Pfenninger, Stefan and Staffell, Iain},
+  title = {{Long-term patterns of European PV output using 30 years of validated hourly reanalysis and satellite data}},
+  journal = {Energy},
+  volume = {114},
+  year = {2016},
+  pages = {1251-1265},
+  doi = {10.1016/j.energy.2016.08.060}
+}
+
+@artcle{Sandwell:2016,
+  author = {Sandwell, Philip and Chambon, Clementine and Saraogi, Amit and Chabenat, Apolline and Mazur, Marek and Ekins-Daukes, Ned and Nelson, Jenny},
+  title = {{Analysis of energy access and impact of modern energy sources in unelectrified villages in Uttar Pradesh}},
+  journal = {Energy for Sustainable Development},
+  volume = {35},
+  year = {2016},
+  pages = {67-79},
+  doi = {10.1016/j.esd.2016.09.002}
+}
+
+@article{Chambon:2020,
+  author = {Chambon, Clementine L. and Karia, Tanuj and Sandwell, Philip and Hallett, Jason P.},
+  title = {{Techno-economic assessment of biomass gasification-based mini-grids for productive energy applications: The case of rural India}},
+  journal = {Renewable Energy},
+  volume = {154},
+  year = {2020},
+  pages = {432-444},
+  doi = 10.1016/j.renene.2020.03.002}
+}
+
+@article{Sandwell:2017b,
+  author = {Sandwell, Philip and Ekins-Daukes, Ned and Nelson, Jenny},
+  title = {{What are the greatest opportunities for PV to contribute to rural development?}},
+  journal = {Energy Procedia},
+  volume = {130},
+  year = {2017},
+  pages {139-146},
+  doi = {10.1016/j.egypro.2017.09.416}
+}
+
+@article{Beath:2021,
```

Comment:
We're getting automated warnings about not being able to find references in the `.md` file, but I feel this is probably ok as the names match up:
```log
Warning:  Citeproc: citation Beath:2021 not found
```

It is however causing errors later in the compilation process:
```log
LaTeX Warning: There were undefined references.
```